src/inapp/inapp.ts

```diff
@@ -505,8 +504,6 @@ export function getInAppMessages(
                 } else {
                   newTabAction();
                 }
-              } else if (link.getAttribute('target') === null) {
-                sameTabAction();
```

Comment:
Is this just to respect browser default instead?

Reply:
Yep that's right. I decided to pull it out of the `manageHandleLinks` method because the conditions for non-handleLinks scenarios slightly differed between Safari and non-Safari.